src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -43,7 +45,13 @@ internal class SonarQubeServiceWrapper : ISonarQubeServiceWrapper
 
         public const string CSharpLanguage = "cs";
         public const string VBLanguage = "vbnet";
-        public static readonly string[] SupportedLanguages = { CSharpLanguage, VBLanguage };
+
+        public static readonly IReadOnlyDictionary<Language, string> LanguageKeys = new ReadOnlyDictionary<Language, string>(
```

Comment:
The old `Language.ServerKey` was removed. This mapping is local to the `SonarQubeServiceWrapper` since the specific string key is a server specific detail which should be hidden (as this might not be stable between plugin versions etc.).